addons/Socket/source/LocalNameServers.c

```diff
@@ -157,7 +157,8 @@ void LocalNameServers_findIps(LocalNameServers *self)
 
 void LocalNameServers_findIps(LocalNameServers *self)
 {
-	char *path = tmpnam(NULL);
+        char *path = NULL;
+	mkstemp(path);
```

Comment:
@acook: I am unsure. I thought the old code captured it, so I preserved it. I may, however, be mistaken.